pages/learn/manage/actions.md

```diff
@@ -90,10 +90,26 @@ The `Delete Device` action is an extremely dangerous action and results in disas
 
 ## Application-specific actions
 
+These actions can be found on the "Actions" menu for each application and apply to the application and all the devices in the fleet.
+
 ### Change Application Type
 
 This option allows you to convert your application to [another type][app-types], as long as the devices in the application meet the {{ $names.os.lower }} version requirements and your account has the appropriate privileges.
 
+### Rename Application
+
+This action allows you to rename your application. This action is only available for new [applications types][app-types] such as Starter, Microservices or Essentials. Its not currently possible to rename `Legacy` or `Classic` applications, you will first need to upgrade your app type.
```

Comment:
tiny nit for consistency:

```suggestion
This action allows you to rename your application. This action is only available for new [applications types][app-types] such as `Starter`, `Microservices` or `Essentials`. Its not currently possible to rename `Legacy` or `Classic` applications, you will first need to upgrade your app type.
```

Reply:
fixed in cb44b287684d0fed3483305b860cf31e2b55f015